src/types/response/index.ts

```diff
@@ -25,7 +25,7 @@ export interface ICheckboxResponse extends IBaseResponse {
   answerArray: string[]
 }
 
-export type ITableRow = string[]
+export type ITableRow = Array<string>
```

Comment:
nit: can we standardise to `string[]`? that's how it's written everywhere else in the codebase

Reply:
edited